src/xunit.performance.analysis/Program.cs

```diff
@@ -287,14 +287,17 @@ private static void WriteTestResultsHtml(Dictionary<string, Dictionary<string, T
         private static string EscapeCsvString(string str)
         {
             // Escape the csv string
-            if(str.Contains("\""))
+            if (str.Contains("\""))
             {
-                str = "\"" + str.Replace("\"", "\"\"") + "\"";
+                str = str.Replace("\"", "\"\"");
             }
-            if (str.Contains(","))
+            int numOfCommas = str.Split(',').Length - 1;
```

Comment:
There's got to be a better way. String.Split is a memory hog.
Is the idea to add a pair of enclosing quotes for each comma?

```C#
using System.Linq;
int numOfCommas = str.Count(ch => ch == ',');
string enclosingQuotes = new string('\"', numOfCommas);
return enclosingQuotes + str + enclosingQuotes;
```